script.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
+// Book Class: Represents a Book
+// eslint-disable-next-line max-classes-per-file
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Store Classs: Hundles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
```

Comment:
- [ ] [OPTIONAL] In order to keep your code clean, short-circuiting can also be used for this. Checkout the code below
` return JSON.parse(localStorage.getItem('books')) || []`